dev/pytest

```diff
@@ -15,9 +15,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+set -o pipefail
+set -e
 
-# Runs both doctests and unit tests by default, otherwise hands arguments over to nose.
+if ! hash pytest 2> /dev/null; then
+    echo "The pytest command was not found. Please install 'pytest' Python package."
+    exit 1
+fi
 
+# Runs both doctests and unit tests by default, otherwise hands arguments over to pytest.
```

Comment:
nit: this comment should be at the previous line of the current line 37 (`if [ "$#" = 0 ]; then`)?